styles/theme/theme.tsx

```diff
@@ -0,0 +1,9 @@
+const primaryTheme = {
```

Comment:
As per my comment on the font's PR, I would suggest to create a folder for example global-styles and add there files for global use. For example, colors file, breakpoints, fonts, space etc.

Since we don't have themes (unless we want themes?? I don't see the point for this project), just use colors: { ....
we don't need the primaryTheme={